### PR TITLE
Docker entrypoint ...

### PR DESCRIPTION
**Pull request by gimler**
* set the correct entrypoint as described in the docs (https://github.com/helm/chart-releaser#docker-for-continuous-integration)
* update alpine to latest stable

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
-FROM alpine:3.12
+FROM alpine:3.13
 
 RUN apk --no-cache add ca-certificates
 
 COPY cr /usr/local/bin/cr
 
-# Ensure that the binary is available on path and is executable
-RUN cr --help
```
Comment (davidkarlsen) [inline Dockerfile:8]: I think it's ok to keep this just to ensure it's ok

Comment (gimler) [reply Dockerfile:8]: ok i can remove the commit.

From my perspective it is more clear what the docker image does with the chmod command. It increase the trust.

Comment (gimler) [reply Dockerfile:8]: @davidkarlsen done

**Review by davidkarlsen — APPROVED** [review]

**Comment by cpanato** [comment]
@gimler thanks for your PR, can you please sign the DCO and squash your commits? thanks!

**Comment by gimler** [comment]
@cpanato i sign the DCO.

I would leave the changes into two separate commits because they are two different things.

**Review by cpanato — APPROVED** [review]

**Pull request merged**